test/get.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 var throws = require('assert').throws;
-var vm = require('vm');
 
 var eq = require('./utils').eq;
 var errorEq = require('./utils').errorEq;
 var S = require('..');
+var T = function() { return true; };
+var F = function() { return false; };
```

Comment:
> Shouldn't `$._UnaryFunction` have warned me though?

I'm not sure about `length` checking. `S.add` is a valid unary function even though it reports its `length` as `2`.